projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss

```diff
@@ -330,6 +367,12 @@
         compact: rem(32px)
     );
 
+    $button-floating-size-indigo: (
+        comfortable: rem(48px),
+        cosy: rem(42px),
+        compact: rem(36px)
```

Comment:
The same applies to this size. It's 32px in the web components library, while it's being changed to 36px here in the compact mode. It's also 40px in the wc library, while it's defined as 42px here. The large/comfy size is the same in both libraries.

Reply:
These changes should reflect in web components too. The Indigo style guide has a small button and a big button corresponding to compact and comfortable, and one button derivative to cozy. According to these updates, we changed the fab.